compiler/testData/diagnostics/tests/nullableTypes/uselessElvis.kt

```diff
@@ -48,6 +48,21 @@ fun testFrom13648() {
 
 fun bar() = <!UNRESOLVED_REFERENCE!>unresolved<!>
 
+fun testUselessElvisLeftIsNull() {
```

Comment:
Make sure to run `DiagnosticTestGenerated` and `PhasedJvmDiagnosticPsiTestGenerated`.